codalab/scripts/initialize.py

```diff
@@ -168,6 +169,24 @@ def insert_data():
     for name, codename in submission_status_set:
         _, _ = CompetitionSubmissionStatus.objects.get_or_create(name=name, codename=codename)
 
+    team_status = [("Denied", "denied", "Team was denied."),
```

Comment:
To simplify things, since this is already ran in docker, I'll just put these above... will do this kind of stuff cleanly in v2!